Formula/nvc.rb

```diff
@@ -0,0 +1,140 @@
+class Nvc < Formula
+  desc "VHDL compiler and simulator"
+  homepage "https://github.com/nickg/nvc"
+  url "https://github.com/nickg/nvc/releases/download/r1.0.0/nvc-1.0.0.tar.gz"
+  sha256 "a60478636268a7d1cad2d1c90e775cfd938199dba6b9c407badbef32753dc30f"
+
+  head do
+    url "https://github.com/nickg/nvc.git"
+
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+  end
+
+  depends_on "llvm" => :build
+  depends_on "check" => :build
+
+
+  ieee_url="http://standards.ieee.org/downloads/1076/1076.2-1996"
```

Comment:
Style nit but would prefer it if you wrote these with spacing:
```ruby
ieee_url = "http://standards.ieee.org/downloads/1076/1076.2-1996"
```
It's easier to read at a glance.